Cesium.externs.js

```diff
@@ -2763,6 +2768,11 @@ Cesium.CustomDataSource.prototype.entities;
  */
 Cesium.EntityCollection = function() {}
 
+/**
+ * @param {string} id
```

Comment:
According to Cesium code, it looks to me that it should be `String|Number`. I opened a tentative fix in Cesium, see https://github.com/AnalyticalGraphicsInc/cesium/pull/5791.

Reply:
Yes makes sense.

Reply:
It appears your previous attempt was the good one. Can you revert to it?

Reply:
Done